gulpfile.js

```diff
@@ -92,11 +92,16 @@ const css_src = [
 ];
 const sass_docs_src = './demo.scss';
 
-const js_build_src = './js/src/*.js';
-const js_bundle_src = (args.skipbootstrapjs) ? js_build_dest + '*.js' : [
-  './node_modules/bootstrap/dist/js/bootstrap.bundle.js',
+const js_transpile_src = './js/src/*.js';
+
+const js_bundle_src = [
+  './node_modules/bootstrap-select/dist/js/bootstrap-select.js',
   js_build_dest + '*.js'
 ];
+if (!args.skipbootstrapjs) {
```

Comment:
Looks everything good. However shouldn't we add `skipbootstrapselect` as a possible argument too?